### DISCUSSION TITLE
4.6.0.1 的弹幕效果意外地感觉还不错

### DISCUSSION THREAD

**Discussion by letsky192** [ywmoyue/biliuwp-lite#63] [General]
![2023-04-29_233056](https://user-images.githubusercontent.com/73059745/235310815-46023fe7-ad35-4f92-9315-7b4f9ba78fa6.png)

如题，这个弹幕效果可以考虑保留。
另外请求增加默认画面比例。因为经常看 CF 视频，会调成 4:3；关掉这个选项卡另开一个视频，还会保持 4:3，并且打开时是黑屏。

**Comment by ywmoyue** [comment]
1. 弹幕效果算bug，我从cz仓库的那个PR Copy代码的时候疏漏了，以后再看要不要加个配置项吧
2. 你说的默认画面比例调成4：3是什么意思？使用什么方式调的？

**Reply by letsky192** [reply]
![2023-04-30_222919](https://user-images.githubusercontent.com/73059745/235358606-c9d638d0-7e08-4e33-a6b7-a516eec356d9.png)

现在的表现是，比如看这个视频的时候调成了 4:3，下次打开另一个视频还会记住是 4:3，并且打开的时候是黑屏，要手动换回默认才出画面。希望能增加设置，不要记住上次调整的画面比例。

**Reply by ywmoyue** [reply]
4.6.13 已修复
